tests/phpunit/CRM/Utils/MoneyTest.php

```diff
@@ -44,4 +46,29 @@ public function subtractCurrenciesDataProvider() {
     );
   }
 
+  /**
+   * Test rounded by currency function.
+   *
+   * In practice this only does rounding to 2 since rounding by any other amount is
+   * only place-holder supported.
+   */
+  public function testFormatLocaleNumericRoundedByCurrency() {
+    $result = CRM_Utils_Money::formatLocaleNumericRoundedByCurrency(8950.3678, 'NZD');
+    $this->assertEquals('8,950.37', $result);
+  }
+
+  /**
+   * Test rounded by currency function.
+   *
+   * This should be formatted according to European standards - . thousand separator
```

Comment:
@eileenmcnaughton this comment doesn't seem to match the text which seems to be using `,` as a thousand separator and `.` as decimal

Reply:
doesn't it     $this->assertEquals('8.950,37', $result);

Reply:
it looks to me to be `$this->assertEquals('8,950.37', $result);`

Reply:
Ah yeah - what I had locally wasn't properly pushed

Reply:
ok test looks right now